nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestCSVExportAction.java

```diff
@@ -104,6 +104,9 @@ public void testSimple() throws Exception {
         assertEquals(10, status.getProcessed());
         assertEquals(10, status.getTotal());
 
+        String url = Framework.getService(DownloadService.class).getDownloadUrl(command.getId());
+        assertEquals(url, status.getResult().get("url"));
```

Comment:
is it possible to retrieve the file from this url ? 
it will be much better than using assertion on internal details.

Reply:
Added a UT